diagnostic_feedback/public/js/src/student/student_quiz.js

```diff
@@ -44,6 +44,11 @@ function StudentQuiz(runtime, element, initData) {
       currentAnswerContainer = ".diagnostic-feedback .current",
       studentViewFormSecSelector = ".diagnostic-feedback .student_view_form section",
       questionId = '.question-id',
+      lessonContentSelector = '.lesson-content',
+      contentSelector = ".content",
+      questionContaner = '.q-container',
+      quizQuestion = '.quiz-question',
+      userAnswers = '.user-answers',
```

Comment:
1. We will fix for your first suggestion in a seperate PR.
2. Fixed
3. These are not our classes. .lesson-content is from apros and .content is from jquery steps.